packages/drivers/odsp-urlResolver/src/test/fluidAppUrlResolver.spec.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-import { strict as assert } from "assert";
+import { strict as assert } from "node:assert";
```

Comment:
Why node:assert? I think we just import from "assert" at other places.

Reply:
With our stricter eslint configurations the explicit specifier is required for node modules. The new package where this ended up must already have a stricter one that we had in the removed package.